runtime/vm/vmthread.c

```diff
@@ -509,7 +509,9 @@ threadParseArguments(J9JavaVM *vm, char *optArg)
 	vm->thrDeflationPolicy = J9VM_DEFLATION_POLICY_ASAP;
 
 	if (cpus > 1) {
-#if defined(AIXPPC) || defined(LINUXPPC)
+#if (defined(AIXPPC) || defined(LINUXPPC)) && !defined(J9VM_ENV_LITTLE_ENDIAN)
```

Comment:
the regression was reported on Linux BE only.  it had better applying this fix to Linux BE only.  something like:
      if LINUXPPC
          if  LITTLE_ENDIAN
                  = 96;
          else
                  = 151;
          endif
    elsif AIXPPC
          = 96;
    ....

Reply:
@zl-wang Do we need to account for the Java version as well? I believe the regression was specifically for the IBM Java 8.

Reply:
there is no Linux BE releases after java8

Reply:
Only IBM builds plinux BE, there are no builds for Java 11+ created at OpenJ9 or Adopt. Also, if the cause was the OMR change and not the compiler change, then it follows the fix applies to all versions.